images/kube-test/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Kubekins 1.8 + gcloud
+FROM gcr.io/k8s-testimages/kubekins-test:1.8-latest
+
+ENV GCLOUD_VERSION 163.0.0
```

Comment:
we can probably unpin this now, right?

Reply:
kubekins-test that runs verify/unit in jenkins doesn't come with gcloud, that's why this is needed.